taskmgr/server/server_test.go

```diff
@@ -0,0 +1,112 @@
+package server
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/MrShanks/Taska/common/task"
+	"github.com/MrShanks/Taska/taskmgr/storage"
+)
+
+func TestNewTaskHandler(t *testing.T) {
+	t.Run("Create new task and check if it's there", func(t *testing.T) {
+
+		// Arrange
+		mockDatabase := storage.InMemoryDatabase{
+			Tasks: []*task.Task{},
+		}
+		newDummyTask := struct {
+			Title string
+			Desc  string
+		}{
+			Title: "new upcomig task",
+			Desc:  "title of a new fancy task",
+		}
```

Comment:
why don't you use the task struct we already have?

Reply:
not sure I totally get what you mean here. If you are referring to our task.task struct, I'm not using that because it has also the ID field that we don't have a the moment we create a new task. I could create a dummy ID but I don't see the scope.

Reply:
You don't need to initialize all the fields of a struct in this case you can do something like this.
```suggestion
		newDummyTask := task.Task {
			Title: "new upcomig task",
			Desc:  "title of a new fancy task",
		}
```